library/src/App.vue

```diff
@@ -212,6 +212,11 @@
               .nio-p.text-primary-darker.nio-bold Forecast Widget
```

Comment:
For the .png image, we should keep static images in the cdn and use the url pointer instead of storing the assets in the codebase for most use cases (especially since this is a library, we don't want to add 93KB to Tacklebox's size) I'll connect tomorrow and we can go over how to do that.